.ci/scripts/build_cslc_s1.sh

```diff
@@ -1,39 +1,13 @@
 #!/bin/bash
 set -e
 
+# Source the build script utility functions
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+
+. "${SCRIPT_DIR}"/util.sh
+
 # Parse args
-while [[ $# -gt 0 ]]; do
-  case $1 in
-    -h|--help)
-      echo "Usage: build_cslc_s1.sh [-h|--help] [-s|--sas-image <image name>] [-t|--tag <tag>] [-w|--workspace <path>]"
-      exit 0
-      ;;
-    -s|--sas-image)
-      SAS_IMAGE=$2
-      shift
-      shift
-      ;;
-    -t|--tag)
-      TAG=$2
-      shift
-      shift
-      ;;
-    -w|--workspace)
-      WORKSPACE=$2
-      shift
-      shift
-      ;;
-    -*|--*)
-      echo "Unknown arguments $1 $2, ignoring..."
-      shift
-      shift
-      ;;
-    *)
-      echo "Unknown argument $1, ignoring..."
-      shift
-      ;;
-  esac
-done
+parse_build_args "$@"
```

Comment:
Not sure what magic is going on here, but it sure saves some lines of code.

Reply:
`"$@"` is bash shorthand for "array of all the command line variables", which includes the name of the script as the first variable (at $0). Basically this is just forwarding all the command line options to the parsing function which has been refactored into util.sh.